app/controllers/organizations_controller.rb

```diff
@@ -143,4 +156,17 @@ def update_organization_params
       .require(:organization)
       .permit(:title)
   end
+
+  def verify_organization_user
+    @removed_user = User.find(params[:user_id])
+    not_found unless @removed_user && current_organization.users.map(&:id).include?(@removed_user.id)
+  end
```

Comment:
If `find` fails this will cause a `404` anyway. So the first part of the conditional is not needed

```ruby
@removed_user = User.find(params[:user_id])
not_found unless current_organization.users.pluck(:id).include?(@removed_user.id)
```